### DISCUSSION TITLE
unable to download files in git actions

### DISCUSSION THREAD

**Discussion by dwang488** [community/community#37229] [Actions]
We are not able to download the screenshot from the github actions test result. For example:
https://github.com/ware2go/React2Go/actions/runs/3318646023
If click on the cypress screenshot, it won't download as usual.

**Comment by airtower-luna** [comment]
The repository you linked seems to be private, so us other users can't check that. I've seen others mention problems with downloading artifacts (e.g. #37230), if it's that you might want to [send a bug report](https://support.github.com/contact/bug-report) to GitHub support, to make sure they're aware of the problem whether they check the forum or not.
